### PR TITLE
when checking a run is complete, look for the local file RTAComplete.…

### PR DESCRIPTION
**Pull request by srl147**
…txt to be consistent with FTP.pm

### CODE REVIEW

File: lib/Monitor/SRS/Local.pm [hunk 1]
```diff
@@ -87,10 +87,10 @@ sub is_run_completed {
 
     my $run_folder = Monitor::RunFolder->new( runfolder_path => $run_path, _schema => $self->schema );
```
Comment (dkj) [inline lib/Monitor/SRS/Local.pm]: Can we get rid of this?

File: lib/Monitor/SRS/Local.pm [hunk 2]
```diff
@@ -87,10 +87,10 @@ sub is_run_completed {
...
 
-    my $netcopy = 'ImageAnalysis_Netcopy_complete_Read'.scalar $run_folder->read_cycle_counts;
+    my $rta = "RTAComplete";
```
Comment (dkj) [inline lib/Monitor/SRS/Local.pm]: I think we should bung this string directly in the regex below.

File: lib/Monitor/SRS/Local.pm (hunk 1 above)
Comment (srl147) [inline lib/Monitor/SRS/Local.pm]: yes

On 19/08/15 19:33, dkj wrote:
>
> In lib/Monitor/SRS/Local.pm 
> <https://github.com/wtsi-npg/npg_tracking/pull/288#discussion_r37450371>:
>
> > @@ -87,10 +87,10 @@ sub is_run_completed {
> >
> >      my $run_folder = Monitor::RunFolder->new( runfolder_path => $run_path, _schema => $self->schema );
>
> Can we get rid of this?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/wtsi-npg/npg_tracking/pull/288/files#r37450371>.
>




-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.

File: lib/Monitor/SRS/Local.pm (hunk 2 above)
Comment (srl147) [inline lib/Monitor/SRS/Local.pm]: yes

I'll make both of these changes and add them to the pul request

Steven

On 19/08/15 19:34, dkj wrote:
>
> In lib/Monitor/SRS/Local.pm 
> <https://github.com/wtsi-npg/npg_tracking/pull/288#discussion_r37450455>:
>
> > @@ -87,10 +87,10 @@ sub is_run_completed {
> >
> >      my $run_folder = Monitor::RunFolder->new( runfolder_path => $run_path, _schema => $self->schema );
> >
> > -    my $netcopy = 'ImageAnalysis_Netcopy_complete_Read'.scalar $run_folder->read_cycle_counts;
> > +    my $rta = "RTAComplete";
>
> I think we should bung this string directly in the regex below.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/wtsi-npg/npg_tracking/pull/288/files#r37450455>.
>




-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.

**Pull request merged**